.github/workflows/node-docker-publish.yml

```diff
@@ -1,54 +0,0 @@
-name: docker-publish
```

Comment:
are you sure it's not in use by any repo ?

Reply:
Overlooked that, thank you. Let's not remove it in this PR then. I will use new mode in `docker-publish` first.

Reply:
Removed in #7